### PR TITLE
Bellman Bar In Kreuzberg (with a MAP!!!)

### PR DESCRIPTION
**Pull request by miccferr**
Oh Powerful Underground Overlord,  please accept this humble PR...

### CODE REVIEW

**Comment by ligthyear** [comment]
Wohhhaaa. With a map? Nice.

One request though. Could you move the map-file into the same folder and keep it with the same name but the `geojson`-exention ("Bellman.geojson")? That would allow us to have this easily separated per item and can easily be copied.

Looking forward!

**Comment by miccferr** [comment]
Moved and Renamed the geojson file

File: Berlin/bars/Bellman.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Cool bar in Kreuzberg (near Görli Park)
+ 
+ ![](http://media.timeout.com/images/resizeBestFit/101284907/660/370/image.jpg)
+ 
+ 
+Photo Credits & Review: [Timeout.com](http://www.timeout.com/berlin/en/bars-and-pubs/bellmans)
+
+
+[Map!](../map.geojson)
```
Comment (ligthyear) [inline Berlin/bars/Bellman.md]: this is now pointing to the wrong url. needs to be updated to ./Bellman.geojson

**Comment by ligthyear** [comment]
great, thank you!

**Pull request merged**